redis/src/cluster_async/mod.rs

```diff
@@ -242,17 +234,49 @@ enum Next<I, C> {
     },
     Reconnect {
         request: PendingRequest<I, C>,
-        addr: String,
+        target: OperationTarget,
     },
     RefreshSlots {
         request: PendingRequest<I, C>,
     },
     Done,
 }
 
+fn get_connection_for_route<C>(
+    initial_route: Option<&Route>,
+    redirect: Option<Redirect>,
+    connections: &ConnectionMap<C>,
+    slots: &SlotMap,
+) -> Option<(String, Option<ConnectionFuture<C>>)>
+where
+    C: Connect + Clone,
+{
+    let addr = initial_route.and_then(|route| slots.slot_addr_for_route(route));
+    get_connection(addr, redirect, connections)
+}
+
+fn get_connection<C>(
```

Comment:
Minor, but can we either rename this or consolidate with the above function, which I believe is the only place it's used? We now have a bunch of functions with the some variation of the name `get_connection` -- this one is particularly irksome :laughing:

Reply:
removed entirely - it was made unnecessary by https://github.com/redis-rs/redis-rs/pull/877